docs/contribution-guide/how-to-contribute.md

```diff
@@ -0,0 +1,280 @@
+# Contributing guidelines
+
+Our goal is to **make kernl development smooth**, **stable** and **open**.
+We can't do this without our user community!
+Please read this document before contributing!
+
+## Code of Conduct
+
+Kernl has adopted the **Contributor Covenant** as its **[Code of Conduct](code-of-conduct.md)**, and we expect project participants to adhere to it. 
+Please read the **[full text](code-of-conduct.md)** so that you can understand what actions will and will not be tolerated.
+
+## Open Development
+
+Kernl uses **[GitHub](https://github.com/ELS-RD/kernl)** as a source of truth. The core team works directly on it. All changes are public.
+
+## Writing and formatting text
+
+In a global way, texts in Issues, PRs and commits should be written **in English**, **with care**, **consistency**, and **clarity**.
+Use (**[Markdown](https://www.markdownguide.org/basic-syntax/)**) to structure your texts, and to make them more readable.
+
+## Development Process
+
+### Ask a question / share your ideas or new features
+
+If you have a question, an idea for a new feature, or an optimization, you can **use the dedicated [discussion forum](todo)**.
+
+??? example "@reviewer: proposal to be validated"
+    ```
+    @team:  what about creating a dedicated discussion forum?
```

Comment:
IMO discussion in Github is nice: the only thing, is that it is sometimes badly used (like it is used as issues and the inverse)

Reply:
You are right. Maybe it is overkill at this point. It is more to avoid discussion issues.